### DISCUSSION TITLE
Dexcom G7 statup issue

### DISCUSSION THREAD

**Discussion by Whitemd2** [NightscoutFoundation/xDrip#3722] [General]
I have been using xDip+ with a Dexcom G6, but am switching to the G7. I have gone thru all the other info on the internet to set it up, but cannot get it to work. I have seen the article that shows a QR code to scan, but it just shows a bunch of text and does nothing. The status page shows that it is missing the QR code. What needs to happen?
![Screenshot_20241018_102722_xDrip+](https://github.com/user-attachments/assets/88fc3bc1-95ad-45c9-bd0e-f8689a4625f2)

**Comment by Navid200** [comment]
You need to scan the QR code as explained in the guide through xDrip.  You need to trigger the scanning from xDrip.
You shouldn't just scan the QR code with your phone.

I have explained the detail here:
https://navid200.github.io/xDrip/docs/Settings/Scan_QR_Code.html

I hope that helps.  if it doesn't, please let me know.

**Reply by Whitemd2** [reply]
Ok, got it scanned. I thought it would auto scan and didn't realize I had to tap the screen to get to scan. But it is still not working.  It looks like it is trying, but shows some errors.
![Screenshot_20241018_110018_xDrip+](https://github.com/user-attachments/assets/8beb0701-dfb9-41db-bae9-838b2f5f5e3f)
![Screenshot_20241018_110402_xDrip+](https://github.com/user-attachments/assets/286d0a11-ee81-465a-855c-bf645dbb511c)

**Reply by Navid200** [reply]
Is there anything else collecting from the G7 like a Dexcom receiver or a t:slim pump?

**Reply by Whitemd2** [reply]
I am using the Dexcom G7 receiver, not the app. I also did a restart of my phone and that's when it started "trying" to connect.  But... good news.... after restarting the collector and waiting almost 30 min, it looks like it is now showing data.

**Reply by Navid200** [reply]
Great
